src/index.ts

```diff
@@ -11,6 +13,9 @@ const kConsole = new KlasaConsole({
 const port = process.env.PORT || 8284;
 const server = gqlServer();
 
+// Sweep the cache daily
+setInterval(sweepOldCache, constants.DAY);
```

Comment:
Bold of you to assume this will run in exactly one day.

Reply:
Implementing redis, which natively supports expire. re-drafted the PR.